documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -0,0 +1,1080 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="new-migrating-virtual-machines-cli_{context}"]
+
+ifdef::vmware[]
+= Migrating a VMware vSphere source provider
+
+You can migrate a VMware vSphere source provider using the CLI.
+
+endif::[]
+ifdef::rhv[]
+= Migrating a {rhv-full} source provider
+
+You can migrate a {rhv-full} ({rhv-short}) source provider using the CLI.
+
+.Prerequisites
+
+If you are migrating a virtual machine with a direct LUN disk, ensure that the nodes in the {virt} destination cluster that the VM is expected to run on can access the backend storage.
+
+include::snip-migrating-luns.adoc[]
+
+endif::[]
+ifdef::ova[]
+= Migrating an Open Virtual Appliance (OVA) source provider
+
+You can migrate Open Virtual Appliance (OVA) files that were created by VMware vSphere as a source provider using the CLI.
+
+include::snippet_ova_tech_preview.adoc[]
```

Comment:
in 2.6 the OVA provider wouldn't be tech preview anymore, would you like to drop it here or in a separate PR?